src/call-session/inbound.ts

```diff
@@ -135,11 +135,15 @@ class InboundCallSession extends CallSession {
       setTimeout(() => resolve(false), 3000);
     });
 
+    const answerHeaders = {
+      "Content-Type": "application/sdp",
+    };
+    if (this.clientId) {
+      answerHeaders["Client-id"] = this.clientId!;
+    }
```

Comment:
You need to tell me why you need this header? Is it required by server?
As I tested, everything works without this header.